data/human/news.txt

```diff
@@ -2150,3 +2150,345 @@ news "graffiti on spaceport wall"
+			"a few pirates watching but not apparently touching it. "
+			"no one there. The area around your ship is almost suspiciously devoid of people. "
+		word
+			"Whatever is going on, your ship appears unscathed."
+
+news "Raiders returning to Mordente-Bridi"
+	location
+		planet "Mordente-Bridi"
+	name
+		word
+			"Repairs"
+	message
+		word
+			"Watching the microgravity section of the spaceport from the bar, you "
+		word
+			"observe "
+			"notice "
+			"spot "
+			"spy "
+		word
+			"a Bastion "
+			"a Firebird "
+			"a Falcon "
+			"a Leviathan "
+		word
+			"returning from a raid, "
+		phrase
+			"Raiders returning to Mordente-Bridi - serious damage"
+			"Raiders returning to Mordente-Bridi - light damage"
+
+phrase "Raiders returning to Mordente-Bridi - serious damage"
+	word
+		"battered with thousands of small perforations. Somebody's turned a mainstay of the pirate arsenal, Gatling Guns, back on them."
+		"with substantial sections of hull heavily battered or missing altogether. Fair guess they didn't have enough anti-missile."
+		"its hull covered in gashes and burns from Plasma Cannons"
+	word
+		". "
+	word
+		"As it comes to a stop in the maintenance bay, the Bridi engineering shuttles begin "
+	word
+		"buzzing "
+		"fluttering "
+	word
+		"around it, assessing the damage and beginning the work on repairs. "
+	word
+		"They'll be working for the rest of the shift on this one."
+
+phrase "Raiders returning to Mordente-Bridi - light damage"
+	word
+		"pockmarked with laser burns but with no apparent serious damage"
+		"lightly dented in a way that suggests that a Proton Gun was to blame"
+	word
+		". "
+	word
+		"As it comes to a stop in the maintenance bay, the Bridi engineering shuttles begin "
+	word
+		"buzzing "
+		"fluttering "
+	word
+		"around it, assessing the damage and beginning the work on repairs. "
+	word
+		"The work won't take the shuttles long."
+
+news "Symbol War"
+	location
+		planet "Gagarin"
+	name
+		word
+			"Guerrilla Symbology"
+	message
+		word
+			"It's an average day in Gagarin's busy spaceport, until "
+		word
+			"your attention is grabbed by an oddity. A "
+			"an unusual commotion catches your attention. A "
+		word
+			"squad "
+			"band "
+			"team "
+		word
+			"of CCOR "
+		word
+			"guerillas "
+			"insurgents "
+		word
+			"have entered the port."
+		word
+			"Almost none "
+			"Few "
+			"Some "
+			"All "
+		word
+			"of them are armed, but it's their other equipment that stands out; paint, plaster, and ladders. They "
+		word
+			"scale the rear wall toward the giant crest of the local government and "
+			"walk along the walls, stopping at each local government crest and% "
+		word
+			"replace the crest, then take down their ladders and leave."
+		word
+			"Spaceport security "
+		word
+			"is noticeably absent the whole scene."
+			"watched but kept their distance."
+			"pointedly ignored it as it happened."
+		replace
+			"%replace the crest" "replacing them"
+			"%" ""
```

Comment:
I've been told that this would fix my suggestion above.
```suggestion
		replace
			"%replace the crest" "replacing them"
			"replacing them, then take down their ladders" "replacing them, then taking down their ladders"
			"%" ""
```